src/pages/sujeongKim/Login/Login.js

```diff
@@ -1,8 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
+import LoginButton from './LoginButton';
 
 const LoginSujeong = () => {
-  return <div>Login</div>;
+  const [userId, setUserId] = useState('');
+  const [userPw, setUserPw] = useState('');
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    navigate('/main-sujeong');
+  };
+
+  return (
+    <main className="login">
+      <div className="borderBox">
+        <h1 className="logo">Westagram</h1>
+        <form className="loginForm">
+          <input
+            type="text"
+            className="loginId loginInput"
+            name="userId"
+            placeholder="전화번호, 사용자 이름 또는 이메일"
+            onChange={e => {
+              setUserId(e.target.value);
+            }}
+          />
+          <input
+            type="password"
+            className="loginPw loginInput"
+            name="userPw"
+            placeholder="비밀번호"
+            onChange={e => {
+              setUserPw(e.target.value);
+            }}
+          />
+          {userId.includes('@') && userPw.length >= 5 ? (
+            <LoginButton
+              btnOn={false}
+              goToMain={goToMain}
+              BtnColor={'loginBtnOn'}
+            />
+          ) : (
+            <LoginButton btnOn={true} goToMain={'/'} BtnColor={''} />
+          )}
```

Comment:
```suggestion
            <LoginButton
              btnOn={userId.includes('@') && userPw.length >= 5 ? false : true}
              goToMain={userId.includes('@') && userPw.length >= 5 ? goToMain : '/' }
              BtnColor={userId.includes('@') && userPw.length >= 5 ? 'loginBtnOn' : '' }
            />
```
중복되는 부분이 있다면 코드에서 최대한 줄이는 것이 효율적인 코드입니다!
현재 피드백으로 드린 코드에도 중복된 부분이 있다는 것을 볼 수 있을텐데요
해당 조건 부분을 변수로 빼서 관리하면 조금 더 효율적으로 관리할 수 있겠습니다!

Reply:
감사합니다 좋은 의견~